config/initializers/simple_form.rb

```diff
@@ -46,7 +46,8 @@
     ## Inputs
     b.use :label_input
     b.use :hint,  wrap_with: { tag: :span, class: :hint }
-    b.use :error, wrap_with: { tag: :span, class: :error }
+    # Commented out until styling kicks in
+    # b.use :error, wrap_with: { tag: :span, class: :error }
```

Comment:
Yup. Perfectly sensible and pithy enough to make it clear why code shouldn't be commented out in any case.